runtime-testsuite/resources/org/antlr/v4/test/runtime/descriptors/ParserExec/ReservedWordsEscaping.txt

```diff
@@ -0,0 +1,35 @@
+[notes]
+https://github.com/antlr/antlr4/issues/1070
+
+[type]
+Parser
+
+[grammar]
+grammar G;
+
+root
+    : {0==0}? continue+ {<writeln("$text")>}
+    ;
+
+continue returns [int return]
```

Comment:
I think we should add an argument here as well, per my other comment. Java definitely generates variables from grammar argument names.